src/modules/shows-api-helpers.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] I would recommend improving the code quality instead of just disabling the linters. Linters help you in improving the code quality so please remove this line and do certain changes to fix the linters.

Reply:
@Dipeshtwis Good point. We will make the changes.

Reply:
We forgot to remove that line of code. 😬